modular_skyrat/modules/modular_vending/code/clothesmate.dm

```diff
@@ -72,6 +72,10 @@
 				/obj/item/clothing/under/tachawaiian/purple = 5,
 				/obj/item/clothing/under/tachawaiian/green = 5,
 				/obj/item/clothing/under/tachawaiian/blue = 5,
+				// BUBBER EDIT START - Tojo's Outfits Clothesmate Definitions
+				/obj/item/clothing/under/red_and_white_collared_outfit = 5,
+				// BUBBER EDIT END
+
```

Comment:
You dont have to do edit comments for skyrat's folders anymore

Reply:
Oh, sweet. Would it be alright to leave this in? Since we've already done quite a few of these. I'd be keeping the separator there anyway as I plan to make more outfits, and it'd be nice to keep it relatively centralized.

Reply:
In the future, change it to something like
```
// Tojo's Outfits
```